Dockerless Installation.md

```diff
@@ -69,7 +69,7 @@ KillSignal=SIGINT
 
 # Connext
 
-TODO: install (waiting for vector)
+Follow the [docs](https://github.com/connext/vector#quick-start).
```

Comment:
This is not good enough imho:

- there are multiple vector stacks
```
Four different Vector stacks are supported:

global: standalone messaging service (+ EVMs in dev-mode)
node: vector node + database
router: vector node + router + database
duet: 2x node/db pairs, used to test one-on-one node interactions
trio: 2x node/db pairs + 1x node/router/db , used to test node interactions via a routing node.
```
Which one do I need?

- also what kind of endpoints do I need to expose for xud?
- and what node should my vector node connect to for swapping with other xud vector clients?

Reply:
1. Docker pull the vector_node image `docker pull connextproject/vector_node:837bafa1`

2. Create `node.config.json` for simnet:
```
{
  "adminToken": "ddrWR8TK8UMTyR",
  "chainAddresses": {
    "1337": {
      "channelFactoryAddress": "0x2eC39861B9Be41c20675a1b727983E3F3151C576",
      "channelMastercopyAddress": "0x7AcAcA3BC812Bcc0185Fa63faF7fE06504D7Fa70",
      "transferRegistryAddress": "0xB2b8A1d98bdD5e7A94B3798A13A94dEFFB1Fe709",
      "TestToken": ""
    }
  },
  "chainProviders": {
    "1337": "http://35.234.110.95:8545"
  },
  "domainName": "",
  "logLevel": "debug",
  "messagingUrl": "https://messaging.connext.network",
  "production": true,
  "mnemonic": "crazy angry east hood fiber awake leg knife entire excite output scheme"
}
```

3. Start node:
```
#!/bin/bash
set -xe
docker run --env VECTOR_CONFIG="$(cat node.config.json)" --env VECTOR_PROD=true --env VECTOR_SQLITE_FILE="/database/store.db" -p "8000:8000" --mount type=volume,source=vector_node_store,destination=/database --name vector_node --rm connextproject/vector_node:837bafa1
```

Reply:
`docker pull` conflicts with a **dockerless** deployment guide. @erkarl 

Agree that it's not good enough, question is do we want to fix that now or as follow up when vector is out and stabl-ish. @michael1011

Reply:
In that case you want the `node: vector node + database` version.

Reply:
Added that